client/lib/posts/actions.js

```diff
@@ -102,8 +100,9 @@ PostActions = {
 	 *
 	 * @param {Number} siteId  Site ID
 	 * @param {Object} options Edit options
+	 * @param {Object} site    Site object
 	 */
-	startEditingNew: function( siteId, options ) {
+	startEditingNew: function( siteId, options, site ) {
```

Comment:
Wondering.
Is there a reason why we're passing both `siteId` and `site` here (and in the other actions as well)?
Can't we pass just `site` and obtain the id via `site.ID` instead?

Reply:
The reason why we sometimes pass both site and siteId is to allow the removing of sitesList in posts to be divided in pull requests. Each one handling a specific part and applying specific pattern. In this specific case, we think we handle all the occurrences in this PR, so I will try to just pass the site.

Reply:
Ha, I usually try the opposite -- to pass only `siteId` as much as possible (instead of `site`), especially when we aren't using many `site` attrs (or any at all). My rationale is parsimony of information passed, and the fact that object props are harder to reason about when it comes to React re-renders.

Reply:
Hi @ockham, Yes normally I also prefer to pass the siteId, and them each component retrieves what it needs using our selectors. But here we are passing site to old stores that don't access the redux state. Before, they retrieved site from SitesList using the id, the solution I found to remove SitesList from them was to pass the site. So they don't need to retrieve it, it was one of the patterns used to remove SitesList from other places.